operator/src/main/java/dev/responsive/k8s/operator/reconciler/KafkaStreamsPolicyPlugin.java

```diff
@@ -101,13 +104,80 @@ public void reconcile(
     final var targetState = maybeTargetState.get();
     LOG.info("target state for app {} {}", appName, targetState);
 
-    if (targetState.getTargetState().isEmpty()) {
+    if (targetState.getTargetState().isPresent()) {
```

Comment:
just to make sure I understand the compat story here - either (1) it gets a target state (old controller) and it performs the applications scaling or (2) it gets the new "action" and performs that.

if it gets no target state and no action (old controller, no action necessary) it just does nothing (instead of previously where it would log)?

makes sense to me

Reply:
yep, that's right